test/prog/dftb+/tests

```diff
@@ -90,6 +90,13 @@ scc/C60                                #? MPI_PROCS <= 4
 analysis/C60_ESP                       #? MPI_PROCS <= 4
 spin/H2O                               #? MPI_PROCS <= 1
 
+#! This block of tests are for code features capable of producing
+#! unphysical results
```

Comment:
As we order the tests according timing, this block is not warranted to stay intact, so probably we should omit the comment. (The directory name dangerous should be scaring enough already...)

Reply:
Removed, but all the test would take about the same time.